packages/aws-cdk-lib/aws-ecr/lib/repository.ts

```diff
@@ -783,7 +782,7 @@ export class Repository extends RepositoryBase {
     // images in the repository and the ability to get all repositories to find the arn needed on delete.
     const provider = CustomResourceProvider.getOrCreateProvider(this, AUTO_DELETE_IMAGES_RESOURCE_TYPE, {
       codeDirectory: path.join(__dirname, 'auto-delete-images-handler'),
-      runtime: CustomResourceProviderRuntime.NODEJS_14_X,
+      runtime: CustomResourceProvider.regionalDefaultRuntime(this),
```

Comment:
I think the term "regional default" leaves open too many unknowns.

- What language is it? (Get Node in there somewhere)
- Who is it for? (This is definitely just for us, people can take a dependency on it, but they shouldn't be scandalized if and when we change it without notice)


`builtInCustomResourceNodeRuntime` ? Bit long...

Reply:
changed to `builtInCustomResourceNodeRuntime` and `builtInCustomResourceProviderNodeRuntime` respectively.